package.json

```diff
@@ -2067,7 +2067,6 @@
         "compile-widgetTester": "cross-env NODE_OPTIONS=--max_old_space_size=9096 webpack --config ./build/webpack/webpack.datascience-ui-widgetTester.config.js --watch",
         "kill-compile-webviews-watchd": "deemon --kill npm run compile-webviews-watch",
         "checkDependencies": "gulp checkDependencies",
-        "preinstall": "npx npm-force-resolutions",
```

Comment:
Removed this, every time i ran `npm i`, the package-lock.json would be different, 
now that we're using npm 8, we can use the native `overrides` section.
Now everytime I run npm i, the package-lock is consistent, doesn't change (ever)